Makefile

```diff
@@ -50,17 +50,19 @@ release_package: java_package
 docu_html: docu_htmlclean docu_check
 	mkdir -p documentation/html
 	$(CP) -vrL documentation/book/images documentation/html/images
-	asciidoctor -v --failure-level WARN -t -dbook -a ProductVersion=$(RELEASE_VERSION) documentation/book/bridge.adoc -o documentation/html/bridge.html
+	# override snippetDir for asciidoctor to get right path to the snippets
+	asciidoctor -v --failure-level WARN -t -dbook -a ProductVersion=$(RELEASE_VERSION) -a snippetDir=${PWD}/documentation/book/api/snippet/ documentation/book/bridge.adoc -o documentation/html/bridge.html
```

Comment:
Why is this `snippetDir` needed? The path should be relative.

Reply:
It was a long story for having this working.
Yes this path is relative, as you can see in the pom.xml the `snippetDir` is defined as `./documentation/book/api/snippet/`. This path works for the openapi-generator which uses the pom.xml folder as basedir. This path doesn't work for asciidoctor (so it can't resolve the includes) because its base dir is the folder of the processed source file. Overriding `snippetDir` on the asciidoctor command line fixes the issue.